ajax.php

```diff
@@ -81,6 +99,7 @@
 				$oDoc = $oObj->Get($aData['sBlobAttCode']);
 				$sFileName = $oDoc->GetFileName();
 				$sIcon = utils::GetAbsoluteUrlAppRoot().AttachmentPlugIn::GetFileIcon($sFileName);
+				$sAttachmentLabel = MakeAttachmentLabel($oDoc, $sObjClass, $iObjId);
```

Comment:
Since the `Attachment` object is available why not do this?

```suggestion
				$sAttachmentLabel = MakeAttachmentLabel($oDoc, $oObj);
```